api/ImageData.json

```diff
@@ -15,10 +15,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "14"
+            "version_added": "3.5"
```

Comment:
This is still later than the api.CanvasRenderingContext2D.createImageData entry. They should match, so can you check which is correct?

Reply:
Our test uses that function to create an `ImageData` instance, and it appears it isn't defined in Firefox 3.  Will correct that data in this PR!